### PR TITLE
fix(front/home): add better home page style

### PR DESCRIPTION
**Pull request by Pierre-Monier**
I think it close this [issue](https://github.com/manga-download/haruneko/issues/15)

### CODE REVIEW

**Comment by Sheepux** [comment]
The home page is not the tutorial i have in mind (but it's the starting point).
I'll have to write the use case in the issues as the titles were only a way to dump what was on my mind.
/LGTM

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "npm": ">=7.6.0"
   },
   "dependencies": {
-    "fs-extra": "9.0.1"
+    "fs-extra": "9.0.1",
+    "svelte-awesome": "^2.3.2"
```
Comment (Sheepux) [inline package.json:14]: Do not use svelte awesome,  use svelte carbon icons so that the icons stays coherents

Comment (Sheepux) [reply package.json:14]: The font awesome code was just a copy/paste legacy from hakuneko :)

Comment (Pierre-Monier) [reply package.json:14]: Ok got it :)

File: src/frontend/sample-svelte/components/Home.svelte [hunk 2]
```diff
@@ -1,46 +1,93 @@
...
-        border-radius: 10%;
-    }
-</style>
+<script>
+    import Search16 from "carbon-icons-svelte/lib/Search16";
+    import Plug16 from "carbon-icons-svelte/lib/Plug16";
+    import Language16 from "carbon-icons-svelte/lib/Language16";
+    import Renew16 from "carbon-icons-svelte/lib/Renew16";
+
+    const iconsStyle = "margin-left: 5px; margin-right: 5px;";
```
Comment (Sheepux) [inline src/frontend/sample-svelte/components/Home.svelte:7]: I think that using class is more consistent with styling than using jsstyles:

```html
<style>
  :global(svg.custom-class) {
    fill: blue;
  }
</style>

<Add16 class="custom-class" />
```

**Review by Sheepux — APPROVED** [review]

**Pull request merged**